pytest_threadleak.py

```diff
@@ -20,6 +22,11 @@ def pytest_addoption(parser):
         'Detect thread leak (default: False)',
         type="bool",
         default=False)
+    parser.addini(
+        'threadleak_exclude',
+        'Regex of thread names to exclude',
+        type="string",
```

Comment:
Looks like “string” is not supported in python 2.7, but we can this the default if you don’t specify the type.